src/types.ts

```diff
@@ -78,6 +78,16 @@ export interface Options {
    */
   imports?: Arrayable<ImportsMap | PresetName | InlinePreset>
 
+  /**
+   * Package presets names.
+   *
+   * Register local installed packages as a preset.
+   *
+   * @default []
+   * @see https://github.com/unplugin/unplugin-auto-import#package-presets
+   */
+  packagePresets?: PackagePreset[]
```

Comment:
```suggestion
  packages?: (PackagePreset | string)[]
```

I guess we could make it simpler